README.md

```diff
@@ -43,27 +27,40 @@ To install Airflow in a developer playground setting (i.e. in our baremetal clus
 # $PASSWORD refers to the password you would like to secure your airflow instance with.
 ./scripts/playground/build.sh -p $PASSWORD
 ```
+### Ad-hoc Performance Tests
```

Comment:
@morenod the docs for running ad hoc testings^ and docs for prebuilt is already in master: https://github.com/cloud-bulldozer/airflow-kubernetes/blob/master/dags/openshift_nightlies/docs/manifest_and_releases.md#prebuilt-clusters